### ISSUE TITLE
Syntax Error Received.

### ISSUE THREAD

**Issue by Space-Cased** [mwielgoszewski/burp-protobuf-decoder#2]
SyntaxError: ("mismatched input 'as' expecting COLON", ('/Applications/Burp/Extensions/protoburp.py', 48, 39, '        except (OSError, RuntimeError) as error:\n'))

```
at org.python.core.PyException.fillInStackTrace(PyException.java:70)
at java.lang.Throwable.<init>(Throwable.java:181)
at java.lang.Exception.<init>(Exception.java:29)
at java.lang.RuntimeException.<init>(RuntimeException.java:32)
at org.python.core.PyException.<init>(PyException.java:46)
at org.python.core.PyException.<init>(PyException.java:43)
at org.python.core.PyException.<init>(PyException.java:39)
at org.python.core.PySyntaxError.<init>(PySyntaxError.java:22)
at org.python.core.ParserFacade.fixParseError(ParserFacade.java:92)
at org.python.core.ParserFacade.parse(ParserFacade.java:184)
at org.python.core.Py.compile_flags(Py.java:1731)
at org.python.core.__builtin__.execfile_flags(__builtin__.java:514)
at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:225)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
at java.lang.reflect.Method.invoke(Method.java:597)
at burp.wrc.<init>(Unknown Source)
at burp.huc.a(Unknown Source)
at burp.buc.run(Unknown Source)
at java.lang.Thread.run(Thread.java:680)
```

**Comment by mwielgoszewski** [comment]
Are you using Jython 2.5 or 2.7? Jython 2.7 added the `as` keyword, which is how raised exceptions can be assigned to variables using modern Python (versus `except Exception, error`).

**Comment by Space-Cased** [comment]
Yes, thank you. It looks like it is functionality that is only available with 2.7+.  Thanks Marcin
